src/containers/redditClone/index.js

```diff
@@ -1,14 +1,24 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import { getPosts, addNewPost } from 'redux/actions/posts.js'
+import { getPosts, addNewPost, updatePostLink, updatePostTitle } from 'redux/actions/posts.js'
 import Reddit from 'components/redditClone/index.js'
 
 export class RedditContainer extends Component {
   componentDidMount() {
     this.props.dispatch(getPosts())
   }
   render() {
-    return <Reddit posts={this.props.posts} loading={this.props.loading} onAddPost={(link, title) => this.props.dispatch(addNewPost(link, title))} />
+    return (
+      <Reddit
+        posts={this.props.posts}
+        currentlyEditingPostLink={this.props.currentlyEditingPost.link}
+        currentlyEditingPostTitle={this.props.currentlyEditingPost.title}
+        loading={this.props.loading}
+        onAddPost={() => this.props.dispatch(addNewPost(this.props.currentlyEditingPost.link, this.props.currentlyEditingPost.title))}
```

Comment:
perhaps this could become `this.props.dispatch(addNewPost())`

you can then use a thunk action with `(dispatch, getState) =>`  you can use `getState` to grab the currentlyEditingPost